### ISSUE TITLE
Unnecessary high contrast override is used for buttons with class ms-Link

### ISSUE THREAD

**Issue by trevorsg** [OfficeDev/office-ui-fabric-react#3985]
#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority: Normal

Products/sites affected: VSTS

#### Describe the issue:

In #2589, you can find many instances where the "color" property is set to "Highlight". Highlight is intended as a background color, so in a dark HC theme, it will be a dark color. 

#### Actual behavior:

The result is that many areas have a dark green text color on a black background (High Contrast Theme 1).
![image](https://user-images.githubusercontent.com/397836/36268050-850ad7c2-1243-11e8-8919-2c5bad50e8a0.png)

#### Expected behavior:

Perhaps the color "HighlightText" was intended? That shows up as white in HC1.
![image](https://user-images.githubusercontent.com/397836/36268036-7709f9d2-1243-11e8-9006-dd93c25f7a60.png)

**Can someone confirm this expected behavior for me? If so I will submit a PR to fix. Thanks.**

**Comment by betrue-final-final** [comment]
I think that maybe @kristoferbrown set those to highlight when he meant to set it to highlightText. Where are you seeing this?

**Comment by trevorsg** [comment]
@betrue-final-final Just look at the PR #2589 and Ctrl+F for "color: Highlight"
